safe/note.go

```diff
@@ -107,7 +107,10 @@ func NewEmptyNote(name string) note {
 }
 
 func editorReadText(existingText string) (string, error) {
-	editor := os.Getenv("EDITOR")
+	editor := os.Getenv("VISUAL")
+	if editor == "" {
+		editor = os.Getenv("EDITOR")
+	}
 	if editor == "" {
```

Comment:
Hi @srstevenson, thanks for the PR!

I agree with @leonklingele on refactoring this into a helper `editorPath` function. Aside from that it looks good to me :)